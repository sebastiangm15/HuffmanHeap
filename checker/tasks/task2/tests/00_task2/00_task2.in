5
4 B4
7 A7
7 C7
12 D12
16 E16
2
00010
1110011
