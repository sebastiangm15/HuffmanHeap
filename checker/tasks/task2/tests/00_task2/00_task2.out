C7 B4 
E16 D12 A7 
